test/e2e/proxy_test.go

```diff
@@ -99,3 +140,63 @@ func (tc *testContext) waitForValidTrustedCAConfigMap() error {
 	}
 	return nil
 }
+
+// getSystemEnvVar returns the value corresponding to the input proxy ENV var as set in the registry
+func (tc *testContext) getSystemEnvVar(addr, variableName string) (map[string]string, error) {
+	command := fmt.Sprintf("Get-ChildItem -Path Env: | Where-Object -Property Name -eq '%s' | Format-List ",
+		variableName)
+	out, err := tc.runPowerShellSSHJob(fmt.Sprintf("get-%s-system-env-var",
```

Comment:
Can you share the documentation for this powershell command?

Reply:
The powershell command is the one above that line it's a generic `Get-ChildIteam -Path` command
https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/get-childitem?view=powershell-7.3
the output is then filtered to get value of the specific ENV var
eg: Get-ChildItem -Path Env: | Where-Object -Property Name -eq 'HTTP_PROXY'